splitgraph/engine/postgres/engine.py

```diff
@@ -1597,13 +1599,6 @@ def _convert_audit_change(
 _KIND = {"I": 0, "D": 1, "U": 2}
 
 
-def _convert_vals(vals: Any) -> Any:
```

Comment:
Note that this is rendered obsolete through `psycopg2.extensions.register_adapter(dict, Json)` from above. In fact it goes a step further and coerces nested objects (i.e. lists of dicts) accordingly.

Let me know what you think about this.